ui/components/app/srp-quiz-modal/QuizContent/QuizContent.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {
+  AlignItems,
+  BlockSize,
+  Display,
+  FlexDirection,
+  JustifyContent,
+  TextAlign,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { Button, Text } from '../../../component-library';
+import Box from '../../../ui/box';
```

Comment:
If you rebase the `Box` component in TypeScript will be available from the `component-library` it would be a 2 line update as the component API remains the same

```suggestion
import { Button, Text, Box } from '../../../component-library';
```